solr/solr-ref-guide/modules/configuration-guide/pages/commits-transaction-logs.adoc

```diff
@@ -62,6 +62,8 @@ It is recommended that this be set for as long as is reasonable given the applic
 A hard commit means that, if a server crashes, Solr will know exactly where your data was stored; a soft commit means that the data is stored, but the location information isn't yet stored.
 The tradeoff is that a soft commit gives you faster visibility because it's not waiting for background merges to finish.
 
+In a TLOG/PULL replica setup, the commit configuration also influences the interval at which the replica is polling the shard leader. You may optionally configure a custom `commitPollInterval`.
```

Comment:
Please start new sentences on new lines.  Not sure if this style choice is stated somewhere.  Goal is reducing diffs and reducing need for line wrap.

Reply:
Fixed.